### PR TITLE
Added initial DE resources

### PR DESCRIPTION
**Pull request by kkneville**
Started a page with some initial resources in German, including one focused on child and adolescent mental health and development. I would like to come back add more when I have time, and parse articles by subject.

### CODE REVIEW

File: DE-_initial_resources [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!-- some initial resources in German about mental health, to be further parsed in a future pull req, this is just the start [kkneville] -->
```
Comment (andy5995) [inline DE-_initial_resources]: Danke @kkneville 

Would you like me to merge this PR, or are you planning to add more commits to this PR?

**Comment by kkneville** [comment]
I would say, go ahead and merge it in case someone comes along and wants to
build on it. I will do my best to get back and do some more work on it in
the coming month - currently in a coding bootcamp, availability is
unoredictable until I finish. But it's a laudable project and I want to
keep contributing!

**Comment by andy5995** [comment]
>currently in a coding bootcamp, availability is unoredictable until I finish.

Good luck with Bootcamp!

>But it's a laudable project and I want to keep contributing!

Vielen Dank, herzlich willkommen 

I'll make a few minor changes to this PR before I merge it. Please note the changes.

After it's merged, I'll put the links on separate pages. The .md files are arranged so that links are easy to find if they need editing.

bis später @kkneville !

File: DE-_initial_resources [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Kinder Unterstützen](https://www.kindergesundheit-info.de/themen/entwicklung/psychische-gesundheit/)<br />
+"tags": ["Kinder", "Entwicklung"]<br />
```
Comment (andy5995) [inline DE-_initial_resources]: I removed "psychische gesundheit" because all links in the knowledge base are about psychische gesundheit :)

**Pull request merged**